src/utils/validation/validate-address.ts

```diff
@@ -0,0 +1,18 @@
+import { decodeAddress, encodeAddress } from '@polkadot/keyring';
```

Comment:
@tomjeatt 
It's my personal opinion so you can make a decision.
I was going to create `helpers` directory at the same level of `constants` and `hooks`. I guess `validate-address.ts` can be included as a helper function.

Reply:
Yep that makes sense - will move it.

Reply:
@tomjeatt 
One more question please, can we harmonize the file name with the function name? I think it could make things e.g. convention a little simpler.